README.md

```diff
@@ -238,6 +238,8 @@ See the [prioritized list of open issues](https://github.com/scikit-hep/awkward-
 
 Support for this work was provided by NSF cooperative agreement OAC-1836650 (IRIS-HEP), grant OAC-1450377 (DIANA/HEP) and PHY-1520942 (US-CMS LHC Ops).
 
+We would also like to thank @erezsh and the developers of [lark](https://github.com/lark-parser/lark) for creating the [lark standalone parser generator](https://github.com/lark-parser/lark/blob/master/lark/tools/standalone.py), that we use in our type parser.
```

Comment:
The authors capitalize "Lark": [their README](https://github.com/lark-parser/lark#readme), so we should do the same.

Let's use Erez Shinan's full name with GitHub handle in parentheses, maybe more like this:

> We also thank Erez Shinan (@erezsh) and the developers of the [Lark standalone parser](https://github.com/lark-parser/lark), which is used to parse type strings as type objects.